src/portuguese.js

```diff
@@ -0,0 +1,43 @@
+
+class PresenteIndicativo {
+  conjugate (word, info) {
+    let ending = [];
+    switch (word.slice(-2)) {
+      case 'ar':
+        ending = info.singular ? ['o', 'as', 'a'] : ending = ['amos', 'ais', 'am'];
+        break;
+      case 'er':
+        ending = info.singular ? ['o', 'es', 'e'] : ['emos', 'eis', 'em'];
```

Comment:
In French we have the subject 'Vous' and 'On' who can be singular or plural. This affect the past participle in French and this is why I need singular and subject. I don't know if there is a similar case like this in Portuguese.